Documentation/ApiOverview/Database/RestrictionBuilder/Index.rst

```diff
@@ -223,3 +228,51 @@ deliver and use an own set of restrictions for own query statements if needed.
    It can be very helpful to debug the final statements created by the `RestrictionBuilder` using
    :php:`debug($queryBuilder->getSQL())` right before the final call to :php:`$queryBuilder->execute()`. Just
    take care these calls **do not** :ref:`end up in production <database-query-builder-get-sql>` code.
+
+
+A very complex example is that you might want to apply one or more restriction/s to only one table. Let's say,
```

Comment:
> A very complex example is that you might want to apply one or more restriction/s to only one table.

I am not sure I understand this sentence. What is complex about applying the restriction to only one table?